Assets/BossRoom/Scripts/Client/Game/Entity/ClientAvatarGuidHandler.cs

```diff
@@ -11,7 +11,7 @@ namespace Unity.Multiplayer.Samples.BossRoom.Client
     [RequireComponent(typeof(NetworkAvatarGuidState))]
     public class ClientAvatarGuidHandler : NetworkBehaviour
     {
-        [SerializeField]
+		[SerializeField]
```

Comment:
This is a tabs to spaces problem.

Reply:
Class not touched by this PR, likely the fact that it's here is git user error again